frontend/packages/network-attachment-definition-plugin/src/components/SubnetHelperText/SubnetHelperText.scss

```diff
@@ -0,0 +1,9 @@
+.info-icon {
```

Comment:
I suggest using more descriptive names to avoid conflict in the future. The easiest way would be to use BEM and add the block name and then use the prefix on these existing names.

Reply:
changed thanks :)